app/models/Business.scala

```diff
@@ -95,4 +95,52 @@ object Business {
     },
     companyNo = map.get(cBiCompanyNo).map(_.toString)
   )
+
+  def fromCSVMap(id: Long, map: Map[String, String]) = Business(
+    id = id,
+    businessName = map.getOrElse(cBiName, cEmptyStr),
+    uprn = map.get(cBiUprn).map(x => java.lang.Long.parseLong(x.toString)),
+    postCode = map.get(cBiPostCode).map(_.toString),
+    industryCode = industryCodeNormalize(map.get(cBiIndustryCode).map(_.toString)),
+    legalStatus = map.get(cBiLegalStatus).map(_.toString),
+    tradingStatus = map.get(cBiTradingStatus).map(_.toString),
+    turnover = map.get(cBiTurnover).map(_.toString),
+    employmentBands = map.get(cBiEmploymentBand).map(_.toString),
+    vatRefs = map.get(cBiVatRefs).map(x => Seq(x.toString)),
+    payeRefs = map.get(cBiPayeRefs).map(x => Seq(x.toString)),
+    companyNo = map.get(cBiCompanyNo).map(_.toString)
+  )
+
+  def toMap(b: Business): Map[String, Any] = Map(
+    cBiName -> b.businessName.toUpperCase,
+    cBiUprn -> b.uprn.orNull,
```

Comment:
Populating a Map with a null value is pretty unusual - particularly in Scala where get will return a Some(null).  Would it not be better to omit the field from the Map when None, so that get returns None?

Reply:
This was in the existing code, it is only used when inserting data into ElasticSearch, so I presume there must be a reason for it? I'll try just using the Option without `.orNull` and see if our `elastic4s` dependancy can handle the conversion of the Option to the Java equivalent (I think elastic4s uses the ElasticSearch Java API).

Reply:
Testing the data load into ElasticSearch with Options rather than nulls results in 400's from ElasticSearch, so I presume the reason for the `.orNull` is because `elastic4s` does not handle the conversion itself.